GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -434,8 +434,11 @@ public override bool TryRunInstaller(InstallActionWrapper installActionWrapper,
 
                 if (!installSuccessful)
                 {
-                    activity.RelatedError($"Could not complete all install actions. The following error was encountered: {localError}");
-                    error = localError;
+                    error = string.IsNullOrEmpty(localError) ?
+                        $"The {currentInstallAction.Name} failed, but no error message was provided by the failing command." :
```

Comment:
If an exception is thrown before `currentInstallAction` is set this we'll have a null reference exception here.